frontend/src/styles/components/_variables.scss

```diff
@@ -9,9 +9,13 @@ $white: #fff;
 $black: #000;
 $red: #d8292f;
 $green: #2e8540;
-$yellow: #fcba19;
+$yellow: #e3a82b;
```

Comment:
is there a reason we switched to this yellow https://developer.gov.bc.ca/Colour-Palette

Reply:
Hi Shawn,

I was given mandate that when making any products that are public facing it is required that we follow the government styling guidelines.  Not sure if this team is exempt from such rules?